src/cbmc/bmc.cpp

```diff
@@ -603,6 +609,11 @@ safety_checkert::resultt bmct::stop_on_fail(
 
       error_trace();
     }
+    if(options.get_bool_option("localize-faults"))
+    {
+      fault_localizationt fault_localization(options);
+      fault_localization(dynamic_cast<bv_cbmct &>(prop_conv), equation, ns);
+    }
     
```

Comment:
It may be irritating that localize-faults only works with one property? Is this in principle meant to deal with multiple traces?